mito-ai/src/Extensions/InlineCompleter/provider.ts

```diff
@@ -326,19 +328,24 @@ export class MitoAIInlineCompleter
   }
 
   private _notifyCompletionFailure(error: CompletionError): void {
-    Notification.emit(`Inline completion failed: ${error.error_type}`, 'error', {
-      autoClose: false,
-      actions: [
-        {
-          label: 'Show Traceback',
-          callback: async (): Promise<void> => {
-            await showErrorMessage('Inline completion failed on the server side', {
-              message: error.traceback ?? 'An unknown failure happened when requesting a completion.'
-            });
+    // Only show the notification if we haven't shown one already
+    if (!this._displayed_completion_failure_notification) {
+      Notification.emit(`Inline completion failed: ${error.error_type}`, 'error', {
```

Comment:
Sweet! While we are at it, can we update this message so it's more understandable. And maybe we can add a `Get Help` button or something that just opens their email with the mailto link and fills in our emails + a subject 

![Screenshot 2025-03-20 at 5 25 00 PM](https://github.com/user-attachments/assets/de170588-434f-4341-8f60-4145d71f4b42)